src/status_im/contexts/chat/messenger/camera/view.cljs

```diff
@@ -99,7 +100,9 @@
 
 (defn bottom-area
   [{:keys [flash camera-type] :as args}]
-  (let [back        #(rf/dispatch [:navigate-back])
+  (let [back        (fn []
+                      (quo.theme/handle-status-bar-screens-per-theme)
```

Comment:
why do we need to use this function here? 🤔 

Seems like a lot to handle if we need to do this in the application in random places 🤔

Reply:
Yes, I think better solution would be to just update status bar color in `:set-view-id-fx` ([ref](https://github.com/status-im/status-mobile/pull/18785#discussion_r1485644048))
This way we will be able to handle all navigation's (screens/modals) and navigation back without manually adding to screens using effects.